mpt/impl/managesieve/core/src/main/java/org/apache/james/mpt/testsuite/CapabilityTest.java

```diff
@@ -19,29 +19,42 @@
 
 package org.apache.james.mpt.testsuite;
 
-import com.google.inject.Inject;
+import java.util.Locale;
+
 import org.apache.james.mpt.host.ManageSieveHostSystem;
+import org.apache.james.mpt.script.SimpleScriptedTestProtocol;
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.util.Locale;
-
-public class CapabilityTest extends ManageSieveMPTTest {
+public abstract class CapabilityTest {
 
-    @Inject
-    private static ManageSieveHostSystem hostSystem;
-
-    public CapabilityTest() throws Exception {
-        super(hostSystem);
-    }
+    public static final String USER = "user";
```

Comment:
I don't know what we would gain, test script hardcode that, factoring is not of much value IMO.